var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -105,6 +109,8 @@ class Qt(Package):
     # depends_on("ogg", when='+multimedia')
 
     use_xcode = True
+    # parallel build fails on Sierra
+    parallel = False
```

Comment:
i will experiment a bit, strangely enough [Homebrew](https://github.com/Homebrew/homebrew-core/blob/master/Formula/qt.rb) appears to be building in parallel and only installing in serial